spec/support/concerns/emphasised_organisations.rb

```diff
@@ -1,12 +1,13 @@
-RSpec.shared_examples "it can have emphasised organisations" do |document_type, example_name|
+RSpec.shared_examples "it can have emphasised organisations" do |document_type, example_name, data_source: :content_store|
   let(:content_store_response) { GovukSchemas::Example.find(document_type, example_name: example_name) }
+  let(:api_response) { fetch_content_item(document_type, example_name, data_source:) }
```

Comment:
I'm not sure about the naming here. Is it always an `api_response`? Also it feels a bit generic.

Reply:
I've changed `api_response` to `content_item` throughout